test/mix_test_watch/paths_test.exs

```diff
@@ -1,7 +1,7 @@
 defmodule MixTestWatch.PathTest do
   use ExUnit.Case
 
-  import MixTestWatch.Path, only: [watching?: 1, excluded?: 2]
+  import MixTestWatch.Path, only: [watching?: 1, watching?: 2, excluded?: 2]
```

Comment:
I think so since watching? would now take a config as well and the only caller to watching?/1 was moved to watching?/2